### DISCUSSION TITLE
Disabling binary upload

### DISCUSSION THREAD

**Discussion by 56KBs** [goreleaser/goreleaser#3268] [Q&A] [answered]
We're using goreleaser to build our Go project & release a Docker image. We only ever make use of the Docker image and was wondering if it was possible via configuration to disable the binary tar.gz being uploaded to the release?

**Comment by caarlos0** [comment, marked as answer]
yes, `release.disable`: https://goreleaser.com/customization/release/

**Reply by 56KBs** [reply]
Thanks - I would like to keep the changelog generation on the release, it's just the artefact I don't want to upload to it.

**Reply by caarlos0** [reply]
ah, that's not really possible right now... feel free to open a feature request though

**Reply by airtonix** [reply]
Looks like it is now (for github): 

```
  # Set this to true if you want to disable just the artifact upload to the SCM.
  # If this is true, GoReleaser will still create the release with the
  # changelog, but won't upload anything to it.
  #
  # Since: v1.11
  # Templates: allowed (since v1.15)
  skip_upload: true
```
